app/src/main/java/com/marcossalto/targetmvd/network/models/UserSignUpSerializer.kt

```diff
@@ -3,11 +3,11 @@ package com.marcossalto.targetmvd.network.models
 import com.squareup.moshi.Json
 
 data class UserSignUp(
-    @Json(name = "username") val userName: String = "",
+    @Json(name = "username") val username: String = "",
     var email: String = "",
     var gender: String = "",
     val password: String = "",
-    @Json(name = "password_confirmation") val passwordConfirmation: String = ""
+    @Json(name = "password_confirmation") val password_confirmation: String = ""
```

Comment:
same: var naming convection.

Reply:
Thanks for your suggestion.
This change was related to KotlinJsonAdapter in Moshi. I fixed it